config/horizon-dashboard.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'prefix' => env('HORIZON_DASHBOARD_PREFIX', 'horizon-dashboard'),
```

Comment:
I do not think that the path should change per environment so please remove the `env`